app.py

```diff
@@ -476,6 +477,22 @@ def email_members():
     return "Emails are being sent!", 200
 
 
+@app.route("/email_list/")
```

Comment:
Why not argument?

Reply:
Most of the other endpoints are written this way. It might be worth investigating if it could improve the code to do it some other way.